ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/templates/namespace.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ''
```

Comment:
If you do all the openshift specific stuff why not create a project instead?

Reply:
/shrug  So many options.  Better to just create a project, sure.